Content.Server/Forensics/Systems/ForensicScannerSystem.cs

```diff
@@ -74,13 +75,15 @@ private void OnDoAfter(EntityUid uid, ForensicScannerComponent component, DoAfte
                     scanner.Fingerprints = new();
                     scanner.Fibers = new();
                     scanner.DNAs = new();
+                    scanner.Residues = new();
                 }
 
                 else
                 {
                     scanner.Fingerprints = forensics.Fingerprints.ToList();
                     scanner.Fibers = forensics.Fibers.ToList();
                     scanner.DNAs = forensics.DNAs.ToList();
+                    scanner.Residues = forensics.Residues.ToList();
```

Comment:
Why aren't you using AddRange?

Reply:
We want to clear out any existing evidence from the scanner on each scan, otherwise it would show old evidence from the last scanned item.